doc/Caliptra.md

```diff
@@ -473,30 +473,30 @@ An example of when an owner must protect assets would be moving from secure mode
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
     <td>Mode selection (Boot Media Integrated and dependent selections)</td>
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
-    <td>Pauser Attribute</td>
+    <td>Pauser attribute</td>
     <td>Integrity</td>
     <td>Proficient</td>
     <td>Glitching</td>
-    <td>Environmental monitoring &amp; protection</td>
+    <td>Environmental monitoring and protection</td>
   </tr>
   <tr>
     <td></td>
     <td>JTAG debug</td>
     <td>Integrity</td>
     <td>Proficient</td>
-    <td>1. Attempt to manipulate RoT execution via JTAG to non POR flows <br>2. Attempt to retrieve device secrets via JTAG when product is field-deployed<br>3. Attempt to retrieve device secrets via JTAG when product is under development/debug</td>
+    <td>1. Attempt to manipulate RoT execution via JTAG to non POR flows <br>2. Attempt to retrieve device secrets via JTAG when product is field-deployed<br>3. Attempt to retrieve device secrets via JTAG when product is under development and debug</td>
```

Comment:
"development or debug"

Reply:
Both lifecycle stages (development and debug) are relevant to this sentence, so "and" is accurate here. Should probably replace to "...while product is being developed and debugged"

Reply:
i like that better